composer.json

```diff
@@ -132,6 +135,10 @@
             "Composer\\Config::disableProcessTimeout",
             "paraunit run --testsuite integration"
         ],
+        "test:mutation": [
+            "Composer\\Config::disableProcessTimeout",
+            "infection --threads=max --only-covered --min-covered-msi=80"
```

Comment:
You can use [`--only-covering-test-cases`](https://infection.github.io/guide/command-line-options.html#only-covering-test-cases) here which will run only test cases claiming to cover the line being mutated. It might make a difference for you assuming you have a lot of I/O in tests.

Reply:
shall we add this before merge?

Reply:
@keradus I would rather wait for infection/infection#1919 🙂.

Reply:
@Wirone released https://github.com/infection/infection/releases/tag/0.27.11

Docs: https://infection.github.io/guide/command-line-options.html#map-source-class-to-test